LayoutTests/imported/w3c/web-platform-tests/css/css-rhythm/block-step-size-none-does-not-establish-indepdendent-formatting-context.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="author" href="mailto:sammy.gill@apple.com">
+<link rel="help" href="https://drafts.csswg.org/css-rhythm/#block-step-size">
+<link rel="match" href="/css/reference/ref-filled-green-100px-square.xht">
+<meta name="assert" content="block-step-size none should not establish an independent formatting context for the box">
+<style>
+div {
+    width: 100px;
+    height: 100px;
+    background-color: green;
+}
+.floating {
+    position: relative;
+    z-index: -1;
+    float: left;
+    background-color: red;
+}
+.block-step-size {
+    block-step-size: none;
+}
+</style>
+</head>
+<body>
+<p>Test passes if there is a filled green square and <strong>no red</strong>.</p>
+<div class="floating"></div>
+<div class="block-step-size"></div>
+</body>
+</html>
```

Comment:
what exactly are we trying to test here? (the more tests, the better, but independent formatting context does not make a box to be a float avoider, BFC does, so maybe just stick to testing BFC behavior? -also, relatively positioned float?)

Reply:
Yes, the goal here was to test that the block container with block-step-size created a BFC. My understanding is that when a block container establishes a formatting context it is a new BFC (along with maybe an IFC), so I think that's what should be happening here and I can make sure it is more clear in the test, but let me know if my logic is wrong. It's from me interpreting this: https://drafts.csswg.org/css-display/#independent-formatting-context

Reply:
yea, that sounds correct (but that's not what the test case's assert says
"<meta name="assert" content="block-step-size values other than none on block level elements cause the box to establish an independent formatting context">")

Reply:
Fixed the test case asserts and file names to make it clear it was testing BFC

Reply:
don't wanna be too nitpicy, but I am still seeing "independent formatting context" in some of those files.

Reply:
No thank you for catching this!